libs/pages/services/src/lib/feature/page-job-create-feature/variable-feature/variable-feature.tsx

```diff
@@ -0,0 +1,95 @@
+import { APIVariableScopeEnum } from 'qovery-typescript-axios'
+import { useEffect, useState } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useNavigate, useParams } from 'react-router-dom'
+import { FlowCreateVariable } from '@qovery/shared/console-shared'
+import { FlowVariableData, VariableData } from '@qovery/shared/interfaces'
+import {
+  SERVICES_CREATION_GENERAL_URL,
+  SERVICES_JOB_CREATION_POST_URL,
+  SERVICES_JOB_CREATION_RESOURCES_URL,
+  SERVICES_URL,
+} from '@qovery/shared/router'
+import { FunnelFlowBody, FunnelFlowHelpCard } from '@qovery/shared/ui'
+import { computeAvailableScope, useDocumentTitle } from '@qovery/shared/utils'
+import { useJobContainerCreateContext } from '../page-job-create-feature'
+
+export function VariableFeature() {
+  useDocumentTitle('Environment Variable - Create Job')
+  const { setCurrentStep, generalData, setVariableData, variableData, jobURL, jobType } = useJobContainerCreateContext()
+  const { organizationId = '', projectId = '', environmentId = '' } = useParams()
+  const navigate = useNavigate()
+  const pathCreate = `${SERVICES_URL(organizationId, projectId, environmentId)}${jobURL}`
+  const [availableScopes] = useState<APIVariableScopeEnum[]>(computeAvailableScope())
+
+  useEffect(() => {
+    return
+
+    !generalData?.name && navigate(pathCreate + SERVICES_CREATION_GENERAL_URL)
+  }, [generalData, navigate, environmentId, organizationId, projectId, pathCreate])
+
+  const funnelCardHelp = (
+    <FunnelFlowHelpCard
+      title={`${jobType === 'cron' ? 'Cron' : 'Lifecycle'} job variables`}
```

Comment:
The jobType must use  `ServiceTypeEnum` can you check everywhere you use regular strings?